build-ps/debian/percona-server-server-8.0.install

```diff
@@ -16,6 +16,11 @@ usr/bin/resolveip
 usr/bin/resolve_stack_dump
 usr/sbin/mysqld
 usr/bin/ps-admin
+
+# confguration files
+debian/extra/mysql.cnf  etc/mysql/
+debian/extra/mysqld.cnf /etc/mysql/mysql.conf.d/
```

Comment:
here it is /etc

Reply:
it should be etc/. anyway I'm reworking this part right now